extension/content.js

```diff
@@ -128,7 +128,7 @@ function addDeleteForkLink() {
 
 	if ($postMergeContainer.length > 0) {
 		const $postMergeDescription = $postMergeContainer.find('.merge-branch-description');
-		const forkPath = $postMergeContainer.attr('data-channel').split(':')[0];
+		const forkPath = `${getUsername()}/${repoName}`;
```

Comment:
@paulmolluzzo I've specifically meant the following case:

You have a personal account **`A`** and you are a part of an organisation **`O`**. You have forked a repo **`B/C`** not to your personal account, but to your organisation **`O`**. So the correct forked repo URL is **`O/C`**. However with that change the extension would still generate **`A/C`** for the forked repo URL.

I don't usually use the GitHub functionality to change accounts on the dashboard, but I sometimes fork repositories to an organisation account when for example I'm doing open-source PR at work and I'm sharing the fork with colleagues.

P.S. Your scenarios are interesting, but just checked and if you change account on the GitHub dashboard that doesn't change account across the whole GitHub, but just the dashboard so I think the scenario above is the only one which we should concerns ourselves with about organisations.

P.P.S. You can create an organisation for free if you want to test things.